server/http.go

```diff
@@ -302,8 +443,82 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, webh
 
 	w.Header().Set("Content-Type", "application/json")
 	if err := json.NewEncoder(w).Encode(post); err != nil {
-		p.API.LogWarn("failed to write response", "error", err.Error())
+		http.Error(w, "failed to write response", http.StatusInternalServerError)
+	}
+}
+func (p *Plugin) slackAttachmentToUpdatePMI(currentValue string, channelID string) model.SlackAttachment {
+	apiEndPoint := fmt.Sprintf("/plugins/%s/api/v1/updatePMI", manifest.ID)
```

Comment:
constant for this

Reply:
`fmt.Sprintf("/plugins/%s/api/v1/updatePMI", manifest.ID)` (value of type string) is not constant.

Reply:
"/plugins/%s/api/v1/updatePMI" create constant for this